doc/api/modules.md

```diff
@@ -61,7 +61,18 @@ module.exports = class Square {
 };
 ```
 
-The module system is implemented in the `require('module')` module.
+The CommonJS module system is implemented in the [`module` core module][].
+
+## Enabling
+
+<!-- type=misc -->
+
+Node.js has two module systems: CommonJS modules and [ECMAScript modules][].
+
+Authors can tell Node.js to use the ECMAScript modules loader
+via the `.mjs` file extension, the `package.json` [`"type"`][] field, or the
+[`--input-type`][] flag. Outside of those cases, Node.js will use the CommonJS
+module loader. See [Determining module system][] for more details.
```

Comment:
Perhaps add a note again that the ES module loader can be accessed in any context via dynamic `import()`.